spec/controllers/v2/schemas_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+RSpec.describe V2::SchemasController do
```

Comment:
does there also need to be a request spec?

Reply:
I guess I'm kinda doing a request spec over a controller spec here, as I guess controller specs are more designed to be "unit-type" tests, so I think I'll switch over 👍